vogon/urls.py

```diff
@@ -91,6 +91,7 @@
     url(r'^project/(?P<project_id>[0-9]+)/edit/$', views.project_views.edit_project, name='edit_project'),
     url(r'^project/create/$', views.project_views.create_project, name='create_project'),
     url(r'^project/$', views.project_views.list_projects, name='list_projects'),
+    url(r'^project/upload/$', views.project_views.upload, name='upload'),
```

Comment:
this should have the project id int he url (like edit project) and be called 'import' instead of 'upload'

Reply:
changed